src/NHibernate.Test/DialectTest/InformixDialectFixture.cs

```diff
@@ -0,0 +1,24 @@
+﻿using NHibernate.Dialect;
+using NHibernate.SqlCommand;
+using NUnit.Framework;
+
+namespace NHibernate.Test.DialectTest
+{
+	[TestFixture]
+	public class InformixDialectFixture
+	{
+		private readonly InformixDialect _dialect = new InformixDialect();
+
+		[Test]
+		public void ToBooleanValueStringTrue()
+		{
+			Assert.That(_dialect.ToBooleanValueString(false), Is.EqualTo("'f'"));
```

Comment:
Shouldn't it be testing `true` here?

Reply:
Yes, it should. Fixed. My shift is over - I'm going to sleep.